tests/checkpointing/test_model_checkpoint.py

```diff
@@ -675,10 +657,9 @@ def test_model_checkpoint_train_time_interval(mock_datetime, tmpdir) -> None:
     )
 
     trainer.fit(model)
-    # Each batch takes 7 sec and we checkpoint every minute. There are 64
-    # batches per epoch, so total time to run is 7*64*2 = 896 sec < 14.96 minutes,
-    # so we should have 14 checkpoints.
-    assert len(os.listdir(tmpdir)) == 14
+    # Each batch takes 7 sec and we checkpoint every minute. There are 64 batches per epoch, so total time to run is
+    # 7*64*2 = 896 sec < 14.96 minutes so we should have 14 checkpoints. +2 for those saved at the end of the each epoch
+    assert len(os.listdir(tmpdir)) == 14 + 2
```

Comment:
The triggers are not exclusive now so we also save at the end of each epoch (2 epochs)